### PR TITLE
Integration Tab clarification cont.

### PR DESCRIPTION
**Pull request by brianteeman**
Continuation of #23711 for the remaining affectecd components
com_contact
com_newsfeeds
com_tags
com_users

### CODE REVIEW

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 3e224aeba44ad7ae433fa1f8bcb27c25486a0cbc<br /><br />Tested with language file from #23711 for component com_contact<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23714">issues.joomla.org/tracker/joomla-cms/23714</a>.</sub>

**Comment by brianteeman** [comment]
@chmst your local branch must have been out of date as that dile has already been merged

**Comment by chmst** [comment]
oops ;) yes

File: administrator/components/com_tags/config.xml [hunk 1]
```diff
@@ -395,6 +395,11 @@
 		label="JGLOBAL_INTEGRATION_LABEL"
 		description="COM_TAGS_CONFIG_INTEGRATION_SETTINGS_DESC"
 	>
+		<field
+			name="integration_sef"
+			type="note"
+			label="JGLOBAL_SEF_TITLE"
```
Comment (Quy) [inline administrator/components/com_tags/config.xml]: Should this be `News Feeds`?

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 3e224aeba44ad7ae433fa1f8bcb27c25486a0cbc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23714">issues.joomla.org/tracker/joomla-cms/23714</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3765c38400d45e097bbf359a02e431695d25ace8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23714">issues.joomla.org/tracker/joomla-cms/23714</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 3765c38400d45e097bbf359a02e431695d25ace8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23714">issues.joomla.org/tracker/joomla-cms/23714</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23714">issues.joomla.org/tracker/joomla-cms/23714</a>.</sub>

**Comment by HLeithner** [comment]
@brianteeman is this series complete?

**Comment by brianteeman** [comment]
@HLeithner it is now ;)

**Comment by HLeithner** [comment]
thx

**Comment by brianteeman** [comment]
thanks

**Pull request merged**